includes.mk

```diff
@@ -0,0 +1,11 @@
+check-docker:
+	@if [ -z $$(which docker) ]; then \
+		echo "Missing \`docker\` client which is required for development"; \
+		exit 2; \
+	fi
+
+check-kubectl:
+	@if [ -z $$(which kubectl) ]; then \
+		echo "Missing \`kubectl\` client which is required for development"; \
```

Comment:
This check seems to be more helpful:

Perhaps a more descriptive docstring that explains to the user how to retrieve kubectl would be helpful.

For example, this docstring from Sphinx is really helpful:

"The 'sphinx-build' command was not found. Make sure you have Sphinx installed, then set the SPHINXBUILD environment variable to point to the full path of the '$(SPHINXBUILD)' executable. Alternatively you can add the directory with the executable to your PATH. If you don't have Sphinx installed, grab it from http://sphinx-doc.org/"